examples/server_updating.py

```diff
@@ -42,20 +44,40 @@
 
 
 async def updating_task(context):
-    """Run every so often and update live values of the context.
+    """
+    This subroutine runs continously beside the server
+    (via asyncio.create_task in run_updating_server).
+    It will increment some values each two seconds.
 
-    It should be noted that there is a race condition for the update.
+    It should be noted that getValues and setValues are not safe
+    against concurrent use.
     """
-    _logger.debug("updating the context")
+
     fc_as_hex = 3
     slave_id = 0x00
     address = 0x10
-    values = context[slave_id].getValues(fc_as_hex, address, count=5)
-    values = [v + 1 for v in values]  # increment by 1.
-    txt = f"new values: {values!s}"
-    _logger.debug(txt)
+    c = 6
```

Comment:
Please use variable names that mean something e.g. count (just like the other variables)